http.go

```diff
@@ -177,6 +177,19 @@ type Response struct {
 	body []byte
 }
 
+// Creates a new Response
```

Comment:
The comments should start with the function name.
And replace Response{} with NewResponse().